cpp/src/DataStorm/DataElementI.cpp

```diff
@@ -634,7 +648,7 @@ DataElementI::forward(const Ice::ByteSeq& inParams, const Ice::Current& current)
 {
     for (const auto& [_, listener] : _listeners)
     {
-        // If there's at least one subscriber interested in the update
+        // If we are forwarding a sample check if at least once of the listeners is interested in the sample.
         if (!_sample || listener.matchOne(_sample, false))
```

Comment:
The `!_sample` check is there because this also forwards other operations. `_sample` is only set for `SubscriberSession::s` when sending a sample.